src/Aggregate/ConfigurableAggregateTranslator.php

```diff
@@ -64,6 +70,7 @@ class ConfigurableAggregateTranslator implements AggregateTranslator
      * @param null|callable $messageToEventCallback
      */
     public function __construct(
+        $versionMethodName = null,
```

Comment:
Please move to second parameter position as the identifierMethodName is the one which is customized most often